.changeset/spotty-parents-beam.md

```diff
@@ -0,0 +1,5 @@
+---
+"@primer/components": patch
```

Comment:
I think of `patch` versions as bug fixes, dependency bumps, etc. `useSafeTimeout` seems like a new feature. I'd call it a `minor` release.

```suggestion
"@primer/components": minor
```